### ISSUE TITLE
[Bug] - update-motd doesn't clear out update notices after upgrade

### ISSUE THREAD

**Issue by stewartsmith** [amazonlinux/amazon-linux-2023#286]
**Describe the bug**
When launching an older AMI, `update-motd` will generate a notice that there are updates available. Once the updates are applied and the instance rebooted, since there is no more updates to apply, the `update-motd` script *should* go and remove the notice of updates being available, but due to a `-s` check in the `update-motd` shell script, if the size of the updated snippet is zero (i.e. there are no updates), it won't overwrite the previous notice, thus leading to the notification that updates are available remaining.

**To Reproduce**
1. Launch AL2023 rc0 or rc1 AMI
2. Apply updates to 2023.0.20230315 release
3. reboot
4. wait until `update-motd` is run, then log in.

**Expected behavior**
The notification of updates being available is cleared.

**Screenshots**
```
    1 Security notice(s)
        1 Important Security notice(s)
Security: kernel-6.1.15-28.43.amzn2023.aarch64 is an installed security update
Security: kernel-6.1.12-19.43.amzn2023.aarch64 is the currently running version

   ,     #_
   ~\_  ####_        Amazon Linux 2023
  ~~  \_#####\
  ~~     \###|
  ~~       \#/ ___   https://aws.amazon.com/linux/amazon-linux-2023
   ~~       V~' '->
    ~~~         /
      ~~._.   _/
         _/ _/
       _/m/'
[ec2-user@<HOSTNAME> ~]$ uname -vr
6.1.15-28.43.amzn2023.aarch64 #1 SMP Thu Mar  9 17:17:24 UTC 2023
```

**Comment by stewartsmith** [comment]
We have a fix, will queue for an upcoming release

**Comment by nmeyerhans** [comment]
The fix is available in the 2023.0.20230322 repos and AMIs
